downloads/swan-lake-release-notes/swan-lake-alpha2/RELEASE_NOTE.md

```diff
@@ -2,41 +2,558 @@
+        - [Email Module Improvements](#email-module-improvements)
+        - [TCP Module Improvements](#tcp-module-improvements)
+        - [UDP Module Improvements](#udp-module-improvements)
+        - [Crypto Module Improvements](#crypto-module-improvements)
+        - [JWT Module Improvements](#jwt-module-improvements)
+    - [Code to Cloud](#code-to-cloud)
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Debugger](#debugger)
+        - [Ballerina Shell REPL - EXPERIMENTAL](#ballerina-shell-repl-experimental)
+    - [Breaking Changes](#breaking-changes)
 
 ### Updating Ballerina
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to Ballerina Swan Lake Alpha2 as follows.
+
+#### For Existing Users
 
-#### For existing users
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+> `ballerina update`
 
-> $ ballerina dist update
+From Swan Lake Alpha1 onwards, the `ballerina` command has to be issued as `bal`. Next, execute the command below to update to Swan Lake Alpha2.
 
-However, you need to use the following commands instead of the above if you have installed:
+> `bal dist pull slalpha2`
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](/downloads/#swanlake).
 
-#### For new users
+#### For New Users
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
 ### Highlights
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+- Support for mapping and error binding patterns in the `match` statement
+- Support for configurable variables of `record` and `table` types
+- Support for decrypting string values using the new `config` lang library
+- Improvements to the HTTP, mime, WebSocket, GraphQL, WebSub, WebSubHub, IO, email, TCP, UDP, crypto, and JWT standard library modules
+- The extension of the Ballerina package distribution file has been changed from `.balo` to `.bala`
+- Improvements to developer tools such as the language server and debugger.
```

Comment:
```suggestion
- Improvements to developer tools such as the language server and the debugger
```

Reply:
No need to repeat "the" and no need for a full stop as these are bullet points and not complete sentences.

Reply:
Yeah, need to remove the period at the end, right?
 ```suggestion
- Improvements to developer tools such as the language server and debugger
```